src/platform/silabs/CHIPPlatformConfig.h

```diff
@@ -118,6 +118,11 @@
 #define CHIP_CONFIG_ICD_CLIENTS_SUPPORTED_PER_FABRIC SL_ICD_SUPPORTED_CLIENTS_PER_FABRIC
 #endif // CHIP_CONFIG_ICD_CLIENTS_SUPPORTED_PER_FABRIC
 
+#ifndef CHIP_CONFIG_ICD_OBSERVERS_POOL_SIZE
+#define CHIP_CONFIG_ICD_OBSERVERS_POOL_SIZE 3
+#endif // CHIP_CONFIG_ICD_OBSERVERS_POOL_SIZE
+static_assert(CHIP_CONFIG_ICD_OBSERVERS_POOL_SIZE >= 3, "ICD Observers pool size must be at least 3");
```

Comment:
Only in the wifi use case no? else it is 2. Probably need a added #ifdef

Reply:
good catch! updated